src/lib.rs

```diff
@@ -86,6 +87,7 @@ pub async fn app(config: &Figment) -> Result<Rocket<Build>> {
             get_content,
             list_content,
             s3_routes::get_content,
+            s3_routes::get_metadata,
```

Comment:
also added in the get_metadata requests as the head requests were failing from the DAPP